core/fullHistory/historyProcessor.go

```diff
@@ -82,54 +83,69 @@ func (hp *historyProcessor) PutTransactionsData(historyTxsData *HistoryTransacti
 
 	epoch := historyTxsData.HeaderHandler.GetEpoch()
 
-	err := hp.hashEpochProc.SaveEpoch(historyTxsData.HeaderHash, epoch)
+	err := hp.hashEpochStorer.SaveEpoch(historyTxsData.HeaderHash, epoch)
 	if err != nil {
-		log.Warn("epochHashProcessor:cannot save header hash", "error", err.Error())
+		log.Warn("epochHashProcessor: cannot save header hash", "error", err.Error())
 		return err
 	}
 
 	for _, mb := range body.MiniBlocks {
-		mbHash, err := core.CalculateHash(hp.marshalizer, hp.hasher, mb)
+		err = hp.saveMiniblockData(historyTxsData, mb, epoch)
```

Comment:
Perhaps log error as well here? Or only here?

Reply:
I cannot do that because in method **saveMiniblockData** I have a loop and i also have to log an error there if appears